packages/plugins/plugin-advertising-id/src/AdvertisingIdPlugin.ts

```diff
@@ -4,25 +4,24 @@ import {
   PluginType,
   SegmentClient,
 } from '@segment/analytics-react-native';
-import { NativeModules } from 'react-native';
+import { getNativeModule } from '@segment/analytics-react-native/src/util';
```

Comment:
Can we export this from the root so that the import is only `@segment/analytics-react-native`?

The reason this is important is because the releases contain CommonJS, Module and TS so that people can use it regardless of their import types. If we use `/src` it forces them to compile TS

Reply:
Oh I see. Got it. Will fix in a sec.